defaults/main.yml

```diff
@@ -34,15 +34,6 @@ rvm1_rvm_version: 'stable'
 # Check and update rvm, disabling this will force rvm to never update
 rvm1_rvm_check_for_updates: True
 
-# GPG key verification, use an empty string if you want to skip this
-# Note: Unless you know what you're doing, just keep it as is
-#           Identity proof: https://keybase.io/mpapis
-#           PGP message: https://rvm.io/mpapis.asc
-rvm1_gpg_keys: 'D39DC0E3'
```

Comment:
Because 
`gpg --keyserver {{ rvm1_gpg_key_server }} --recv-keys {{ rvm1_gpg_keys }}'`
is repalced by 
`curl -sSL https://rvm.io/mpapis.asc | gpg --import`
The gpg keys are not a problem anymore.

So, if `409B6B1796C275462A1703113804BB82D39DC0E3` is changed to something else. The changes are reflected in gpg key file https://rvm.io/mpapis.asc.

Therefore, the need of rvm1_gpg_keys: '409B6B1796C275462A1703113804BB82D39DC0E3' is removed.